.travis.yml

```diff
@@ -2,18 +2,7 @@ sudo: false
 language: python
 matrix:
   include:
-    - python: "3.5"
```

Comment:
Did you mean to delete this sphinx build? @jonathansick is it being used?

Reply:
Oh I totally forgot about that Sphinx build. Yes, it can be deleted. Thanks!